sasdata/data_util/manipulations.py

```diff
@@ -937,7 +956,7 @@ def _agv(self, data2D, run='phi'):
             # Note that their values must not be altered, as they are used to
             # determine what points (also in the range 0, 2pi) are in the ROI.
             if phi_min > phi_max:
-                binning = Binning(phi_min, phi_max + 2 * np.pi, self.nbins, self.base)
+                binning = Binning(phi_min, phi_max + 2 * math.pi, self.nbins, self.base)
```

Comment:
Replace with `TwoPi` instead here and below? Would be easy if the definition was imported or defined outside a class/method/function.

Reply:
Replaced all occurences in the file with Pi and TwoPi.